### ISSUE TITLE
No progress?

### ISSUE THREAD

**Issue by robertradar** [affinelayer/pix2pix-tensorflow#139]
I use Windows 7 and Python 3.5 with TensorFlow 1.4.0 (there is no 1.4.1 for Windows).

When I run the code in training mode, the last two outputs are
"INFO:tensorflow:Starting queue runners." and
"parameter_count = 57190084".

The output file options.json was created, but there is no further progress/no displayed progress.

What is going wrong here?

**Comment by tonmoyborah** [comment]
In my experience, it takes a long time to start. Did you wait enough?

**Comment by robertradar** [comment]
I waited more than 1 hour. I think this should be enough to see some display output.

Meanwhile I use the torch implementation of pix2pix.

**Comment by julien2512** [comment]
@robertradar I encounter problem in a same way using not enough video ram memory on my on pix2pix fork. I changed my code to use 128*128 instead of 256 (there is several things to do), then it worked.
